src/website/views/utilities/text.pug

```diff
@@ -4,48 +4,36 @@ description: Use text utility classes to control sizing, color, alignment and mo
 styles: [assets/styles/show-example]
 ---
 
+- var tabs = {'xs' : '12px (0.75rem) font size with 20px (1.25rem) line-height','sm' : '13px (0.8125rem) font size with 20px (1.25rem) line-height','md' : '14px (0.875rem) font size with 24px (1.5rem) line-height', 'lg' : '16px (1rem) font size with 24px (1.5rem) line-height', 'xl' : '18px (1.125rem) font size with 28px (1.75rem) line-height'};
+
 h2 Text size
-h3 Base
-p.-text 14px (0.875rem) font size with 24px (1.5rem) line-height
+p.-text
+  | Text support the following sizes: Extra Small (xs), Small (sm), Medium (md), Large (lg), and Extra Large (xl).
 .example.-mb--3
-  p.-text.-m--3
-    | Lorem ipsum dolor sit amet, consectetur adipiscing elit. Vestibulum lacus lacus, dictum quis mauris vel,
-    | commodo condimentum odio. Praesent lacus metus, vehicula at orci ac, fringilla mollis mauris.
-  :code(lang="html")
-    <p class="-text"></p>
+  .-p--3
+    each text, size in tabs
+      div.chi-col.-w--12.-pb--2
+        p.-text--bold= size
+        p= text
+        p(class=`-text--${size}`)
+          | Lorem ipsum dolor sit amet, consectetur adipiscing elit. Vestibulum lacus lacus, dictum quis mauris vel,
+          | commodo condimentum odio. Praesent lacus metus, vehicula at orci ac, fringilla mollis mauris.
+    :code(lang="html")
```

Comment:
Minor bug - need to outdent. Code snippet is rendering in `.-p--3` div instead of `.example` div